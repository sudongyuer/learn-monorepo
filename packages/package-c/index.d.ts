declare function sayHelloC(): void;
export { sayHelloC };
