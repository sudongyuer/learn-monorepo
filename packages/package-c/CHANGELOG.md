# @haiyaotec/package-c

## 4.0.1

### Patch Changes

- patch

## 4.0.0

### Major Changes

- update packages name
