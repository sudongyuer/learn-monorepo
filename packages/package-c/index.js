import { sayHelloB } from "package-b";
function sayHelloC() {
    sayHelloB();
    console.log('C');
}
sayHelloC();
export { sayHelloC };
