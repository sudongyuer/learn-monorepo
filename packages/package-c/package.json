{
  "name": "@haiyaotec/package-c",
  "version": "4.0.0",
  "description": "",
  "main": "index.js",
  "type": "module",
  "scripts": {
    "test": "echo \"Error: no test specified\" && exit 1",
    "build": "tsc"
  },
  "author": "sudongyuer",
  "license": "MIT",
  "dependencies": {
    "package-b": "workspace:*"
  }
}
