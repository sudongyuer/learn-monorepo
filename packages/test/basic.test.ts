import {assert,describe,expect,it} from 'vitest'
import {sayHelloC} from 'package-c'


describe('',()=>{
  it('test',()=>{
    sayHelloC()
  })
})
