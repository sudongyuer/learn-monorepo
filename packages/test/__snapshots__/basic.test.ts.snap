// Vitest Snapshot v1

exports[`snapshot 1`] = `undefined`;
